src/services/language.py

```diff
@@ -13,7 +13,9 @@
 elif language == "zh_cn":
     from data.zh_cn.fonts_description import fonts_description
     from data.zh_cn.text import *
-
+if language == "es":
```

Comment:
```suggestion
elif language == "es":
```

No need to evaluate if language is equal to `es` if it's already equal to `en` for example